runtime/lua/man.lua

```diff
@@ -432,7 +433,7 @@ local function verify_exists(sect, name, silent)
 
   if not silent then
     -- finally, if that didn't work, there is no hope
-    man_error('no manual entry for ' .. name)
+    return nil
```

Comment:
Why has this line been changed?

Reply:
Handled by `open_page()` now (see below).

Not sure of the full implications, but in theory it makes sense for verify_exists() to let the caller decide if an error should be raised.

Reply:
No the silent flag can now be dropped if we don't want to error in this function. I would expect a function prefixed with `verify_` to raise errors. Should this be renamed as it is now just a higher level version of `get_path()`.

Reply:
Renamed it to `search_for_path`, however I am not sure about this as it adds some ambiguity because now there are three functions with simillar names: `get_path`, `get_paths` and `search_for_path`.

Reply:
could be an area worth refactoring but let's do that in a followup PR